lib/Language/Souffle/Interpreted.hs

```diff
@@ -103,7 +116,7 @@ runSouffleWith cfg (SouffleM m) = runReaderT m cfg
 -- | A datatype representing a handle to a datalog program.
 --   The type parameter is used for keeping track of which program
 --   type the handle belongs to for additional type safety.
-newtype Handle prog = Handle (IORef HandleData)
+newtype Handle prog = Handle (IORef HandleData, IORef (Maybe IO.Handle), IORef (Maybe IO.Handle))
```

Comment:
Tuple is not needed, we can change `newtype` into `data`? Or is there another reason for this?

Reply:
Changed.